app/components/record-filter.js

```diff
@@ -1,8 +0,0 @@
-import { computed } from '@ember/object';
-import Component from '@ember/component';
-export default Component.extend({
```

Comment:
This component is replaced with a simple comparison in the template: https://github.com/emberjs/ember-inspector/pull/1034/files?file-filters%5B%5D=.hbs&hide-deleted-files=true#diff-97c2c032aa308abb534f8894b734348eR18